docs/devel/feature-flow-issue-template.md

```diff
@@ -0,0 +1,63 @@
+Thanks for wanting to add a feature to Kubernetes!  You will be responsible for guiding
+your feature through completion, and asking the right people for approvals.  
+
+Large features typically go through three stages: [Alpha, Beta, and Stable](https://github.com/kubernetes/kubernetes/blob/master/docs/api.md#api-versioning)
+Each stage requires various approvals from various teams.
+
+You can use the checkboxes below to
+track your progress.  You can delete the text above once you have read it.  Please keep the checklist.
+
+- Before Alpha
+    - [ ] Design Approval
+      - [ ] Propose your feature.  See [Advice for proposing features].  **PROPROSAL-ISSUE-NUMBER**
+      - [ ] Start with a < 1 page description of design in an issue comment below this. [advice for design description]  **1PAGER-ISSUE-NUMBER**
+      - [ ] Initial design review  		     [advice for getting a review from a team]  **DESIGN-REVIEW-ISSUE-NUMBER**
```

Comment:
I just realized that we could use docs/proposals/OWNERS to ensure proposals are reviewed by appropriate people.  I don't think we need a separate line item for "Initial Design review".